test/parallel/test-bootstrap-modules.js

```diff
@@ -11,4 +11,4 @@ const list = process.moduleLoadList.slice();
 
 const assert = require('assert');
 
-assert(list.length <= 73, list);
+assert(list.length <= 76, list);
```

Comment:
I'm not really in favor of adding 3 more modules in our bootstrap.

Reply:
Agree, will try to cut this down :-)

Reply:
Made this a little bit better.